app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -260,9 +260,32 @@ def transform_metrics(img_metrics)
         end
 
         def parse_date(year:, month:, day:)
-          if year.to_i.positive? && month.to_i.positive? && day.to_i.positive?
-            Date.new(year.to_i, month.to_i, day.to_i).to_s
```

Comment:
Since `Date.new` throws a specific error (ArgumentError), can we just call the constructor and add a rescue clause? It would let us simplify all the code below and leverage the stdlib's knowledge of the calendar. 

I think that the knowledge of which field is wrong is not necessarily useful because we have to redact it, so IMO I would remove that and just log that it was invalid

Reply:
Sounds good. Done in 5408a867b